packages/jupyter-chat-extension/src/__tests__/jupyter_chat_extension.spec.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Example of [Jest](https://jestjs.io/docs/getting-started) unit tests
+ */
+
+describe('@jupyter/chat', () => {
+  it('should be tested', () => {
```

Comment:
Should we remove these jest tests in the extension? Galata tests are probably enough